src/javascripts/menu.js

```diff
@@ -37,7 +37,7 @@ var template = [
       // { label: "Reload Window", accelerator: "CmdOrCtrl+R", click: reload },
       {
         label: "Toggle Full Screen",
-        accelerator: "Cmd+Ctrl+F",
+        accelerator: "CmdOrCtrl+F",
```

Comment:
I believe non-Mac OSes don't have a CMD key, could be wrong, but was why I changed this

Reply:
This is a good catch that requires more testing on Windows. However I think changing the full screen shortcut to `CmdOrCtrl+F` will interfere with the `Cmd+F` find function on Mac and `Ctrl+F` find function on Windows. 

Toggling full screen for MacOS apps are typically done by pressing three buttons `Cmd+Ctrl+F`, which is what my rationale was for the original mapping. Perhaps making the Windows equivalent be `Ctrl+Alt+F` work as an intuitive parallel? What do you think?

Reply:
That works, basically just really putting it up to whatever is good, can be changed to whatever is easiest for all OSes